pyomo/solvers/tests/mip/test_convert.py

```diff
@@ -137,7 +140,11 @@ def test_mod_lp1(self):
                           "enabled but not available: '%s'" % str(err))
             return
         self.assertTrue(ans[0][0].endswith("glpsol.lp"))
-        self.assertFileEqualsBaseline(ans[0][0], currdir+"test2_convert.lp", filter=filter)
+        with open(ans[0][0], 'r') as f1, open(currdir+"test2_convert.lp") as f2:
+            for line1, line2 in zip(f1, f2):
```

Comment:
This should probably use `itertools.zip_longest` so that differing length files cause errors.